src/util/createServerRootMixin.js

```diff
@@ -205,7 +187,10 @@ function augmentInstantSearch(
       results,
       scopedResults: parent.getScopedResults().map(result =>
         Object.assign(result, {
-          results: search.__initialSearchResults[result.indexId],
+          results: new SearchResults(
+            new SearchParameters(search._initialResults[result.indexId].state),
+            search._initialResults[result.indexId].results
+          ),
```

Comment:
this is a little inefficient, maybe at this point they are already set on the parent?